Fly or Die/Assets/LedgeScript.cs

```diff
@@ -20,8 +23,11 @@ void Update()
         // Move left
         transform.Translate(Vector2.left * speed * Time.deltaTime);
 
-        // Collider only enabled when below player
-        float bottomOfPlayer = player.GetComponent<Collider2D>().bounds.min.y;
-        GetComponent<Collider2D>().enabled = transform.position.y <= bottomOfPlayer ? true : false;
+        // Collision ignored when ledge is above player
+        float bottomOfPlayer = playerColliders[0].bounds.min.y;
```

Comment:
This is better than the previous impl because it doesn't totally disable the ledge's collider. It instead just make's the physics engine ignore collisions between _specifically_ the ledge and player's colliders. Non-player object collisions with the ledge will still be enabled.